backend/lambda/function.mjs

```diff
@@ -1,13 +1,16 @@
 import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
 import {
     DynamoDBDocumentClient,
-    ScanCommand,
     QueryCommand,
     PutCommand,
     GetCommand,
-    DeleteCommand,
 } from "@aws-sdk/lib-dynamodb";
 import crypto from "crypto";
+import { request } from "http";
+
+/* TODO Refactor Ideas
+ * Separate files for different APIs?
+*/
 
```

Comment:
What do you mean by:

> Separate files for different APIs?

Reply:
Move all the code per API into its own file. One for "Create" one for "Get All" one for "Get" one for "Update". So that this one doesn't become gigantic

Reply:
Ah I get it! That might be good 👍